spec/presenters/two_factor_auth_code/piv_cac_authentication_presenter_spec.rb

```diff
@@ -25,7 +25,46 @@ def presenter_with(arguments = {}, view = ActionController::Base.new.view_contex
         app: content_tag(:strong, APP_NAME))
     end
 
-    it { expect(presenter.help_text).to eq expected_help_text }
+    context 'with AAL3 required, and only one method enabled' do
+      let(:expected_help_text) do
+        t('instructions.mfa.piv_cac.confirm_piv_cac_only_html')
+      end
+      it 'finds the PIV/CAC only help text' do
+        aal3_policy = instance_double('AAL3Policy')
```

Comment:
I would consider stubbing this with this syntax, to make it read a bit more easily:

```ruby
aal_policy = instance_double('AAL3Policy', aal_required?: true, multiple_aal3_configurations?: false)
```